spec/factories.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    name 'misha'
+    surname 'shevjh'
+    phone_number '+380977588109'
```

Comment:
Does it pass the validation? I see you have `numericality: true` but here are adding `+`